java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1303,13 +1303,13 @@ public Integer delete(User loggedInUser, String advisoryName)
     }
 
     /**
-     * Publishes an existing (unpublished) errata to a set of channels
+     * Publishes an existing errata to a set of channels
      * @param loggedInUser The current user
      * @param advisory The advisory Name of the errata to publish
      * @param channelLabels List of channels to publish the errata to
      * @return the published errata
      *
-     * @xmlrpc.doc Publish an existing (unpublished) errata to a set of channels.
+     * @xmlrpc.doc Publish an existing errata to a set of channels.
```

Comment:
do we still need this method? shouldn't it be removed entirely?

Reply:
Not really.

Publishing a patch to a channel can happen either when the patch has never been published before (unpublished, the case we are removing with this PR and its sister #2280) or when the errata was already published - just in a different channel. The second use case is not expected to go away - and it's actually essential to CLM, for example, when promoting content from a channel (say "test") to another (say "production").

We could come up with another word, but I think publish still makes sense, and in so doing we don't break existing scripts, documentation, etc.

Reply:
I understand that. The thing is not clear to me is: it seems this method was used (AFAIK) _only_ to published "unpublished" erratas. We don't have "unpublished" erratas anymore.

So, either this method should go away now, or this method was also used before to also publish "published" erratas.

Reply:
This is an XMLRPC method, used by spacecmd among others. So already before this PR spacecmd happily passed in published errata IDs, and those worked fine.

Basically here I corrected the docs because code below did work and was already actively used for published errata as well. It was only documentation being out-of-date - maybe initially this method really only worked / was intended for unpublished errata. That's just not the case today.

Does this clarify?

Reply:
Perfect, understood.Thanks